### PR TITLE
Matvei concrete notes

### PR DESCRIPTION
**Pull request by whatcoloris**
reversed changes to index and project-description pages, i like this branch now. :)

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <div class="collapsible-body">
           <ul>
             <li class="waves-effect {% if page.permalink contains 'guidelines' %}active{% endif %}">
-              <a href="/guidelines/">Guidelines</a>
+              <a href="/guidelines/">Guidelines &amp; FAQ</a>
```
Comment (edwardsharp) [inline _includes/nav.html]:
```suggestion
              <a href="/guidelines/" class="overflow">Guidelines &amp; FAQ</a>
```

need to add an `overflow` class to this `<a>` to get its text to wrappp

**Review by edwardsharp — CHANGES_REQUESTED** [review]
can you fix the `Guidelines & FAQ` nav item? 

also, you've got sub-module changes in this PR that (i'm pretty sure) have already been merged. you should be able to rebase this branch from master to make these disappear (`git rebase master`)

File: _includes/nav.html (hunk 1 above)
Comment (whatcoloris) [reply _includes/nav.html]: changes made!

**Review by edwardsharp — APPROVED** [review]
w00t :shipit:

**Pull request merged**